src/services/EdFi.DataManagementService.Api.Tests/ApiSchemaBuilder.cs

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Text.Json.Nodes;
+using EdFi.DataManagementService.Api.ApiSchema;
+using Microsoft.Extensions.Logging.Abstractions;
+
+namespace EdFi.DataManagementService.Api.Tests;
+
+/// <summary>
+/// This class provides a fluent interface for building an ApiSchema suitable for unit testing,
+/// allow tests to focus on scenarios without getting bogged down in JSON authoring
+/// </summary>
+public class ApiSchemaBuilder
```

Comment:
Like MetaEd's MetaEdTextBuilder but for the DMS. A fluent interface for building ApiSchema.json for test scenarios.

Reply:
I should note that this is just enough to support these first tests. As DMS expands its usage of the ApiSchema.json, we'll add test support here for those newly used elements.